src/components/OneToDo/OneToDo.jsx

```diff
@@ -71,17 +71,21 @@ function OneToDo({ toDo, editToDo, deleteToDo, index }) {
   return (
     <div className="one-to-do">
       <div className="left-side">
-        <Checkbox
-          className="left-side-checkbox"
-          checked={toDoStatus}
-          onChange={handleToDoStatus}
-        />
+        {!editingToDo ? (
```

Comment:
update this to conditionally toogle the input and typography based on toogleEditBtn state